test/e2e/nstemplatetier_test.go

```diff
@@ -100,9 +100,9 @@ func TestUpdateNSTemplateTier(t *testing.T) {
 	// finally, all user namespaces are verified.
 	// So, in this test, we verify that namespace resources and cluster resources are updated, on 2 groups of users with different tiers ;)
 
-	count := 2*MaxPoolSize + 1
+	count := 2*1 + 1
```

Comment:
is this a valid change?

Reply:
that was not intended to be here, I'll revert that one

Reply:
reverted in f1497e6